### ISSUE TITLE
Fetch choices from an endpoint

### ISSUE THREAD

**Issue by tiagostutz** [bancodobrasil/stop-analyzing-embed#94]
Add a `useEffect` to the `useChoiceBoardModel.ts` that fetches the initial choices from an endpoint and set the initial options from that.

Tip: this endpoint could be the mock server endpoint

**Comment by priya-pri** [comment]
Hi ,Can you assigned me this?

**Comment by tiagostutz** [comment]
HI @priya-pri ! 👋🏻 
Welcome to stop analyzing! 

Sure, I've just assigned the issue to you. If you need some assistance, please let me know.!

Thanks for your help! =)

**Comment by priya-pri** [comment]
> HI @priya-pri ! 👋🏻
> Welcome to stop analyzing!
> 
> Sure, I've just assigned the issue to you. If you need some assistance, please let me know.!
> 
> Thanks for your help! =)

Thanks @tiagostutz

**Comment by YASH3166** [comment]
HI @YASH3166

Sure, I've just assigned the issue to you. If you need some assistance, please let me know.!

**Comment by mpirescarvalho** [comment]
Is this done? I would like to contribute

**Comment by tiagostutz** [comment]
> Is this done? I would like to contribute

Hi @mpirescarvalho ! 👋🏻 

Are you still interested in tackling this issue?

**Comment by TD-17** [comment]
Will you please assign this issue to me ?

**Comment by tiagostutz** [comment]
Sure, just assigned! Go ahead.!
Thanks!

**Comment by adam-p-git** [comment]
May I take on this one as well?

**Comment by tiagostutz** [comment]
Sure @adam-p-git !
Go ahead. Just assigned to you.!

**Comment by adam-p-git** [comment]
@tiagostutz Would it be okay if I rewrite fetching logic with the help of react-query and hooks? Also I have a suggestions to change /choice endpoint into /choices, I think that is more suitable.

**Comment by tiagostutz** [comment]
Absolutely, go ahead. You can implement and we can discuss this, if needed, in the PR.

Awesome.!

**Comment by KuldeepShakya1159** [comment]
Hey ,I would like to contribute in this issue

**Comment by Saksham-garg** [comment]
@tiagostutz  Hey, I would like to contribute in this issue, Please let me know if this needs to be done.

**Comment by RUSHIM24** [comment]
hey , I this is my first issue is open source i want to contribute in this , can you assign to me

**Comment by shivam-puri** [comment]
Hey Can i help? I wanna contribute!
